components/micro-gateway-core/src/main/ballerina/src/gateway/handlers/api_key_handler.bal

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/stringutils;
+
+# Representation of the api key validating handler
+#
+# + apiKeyProvider - The reference to the jwt auth provider instance
+public type APIKeyHandler object {
+
+    *http:InboundAuthHandler;
+    public APIKeyProvider apiKeyProvider;
+
+    public function __init(APIKeyProvider apiKeyProvider) {
+        self.apiKeyProvider = apiKeyProvider;
+    }
+
+    # Checks if the request can be authenticated with the Bearer Auth header.
```

Comment:
Correct the comment

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/932/commits/80d66417f6156d77e0396413735e126f5b839df6